### DISCUSSION TITLE
QSL-Card image upload

### DISCUSSION THREAD

**Discussion by KeyOz** [magicbug/Cloudlog#833] [Ideas / Feature Requests]
It would be good if you could select a QSL-Card image that has already been uploaded for a callsign. This is good if several QSOs have been confirmed with one QSL card, so that they do not have to be uploaded separately for each QSO.

**Comment by AndreasK79** [comment]
I've implemented a way to add more QSOs to an uploaded QSL Card. I've added a new button under QSO -> View QSL.

**Comment by ofadam** [comment]
Would there be a way to download the eQSL virtual card and have it stored in a similar way?

**Reply by magicbug** [reply]
Eqsl cards can already be viewed and cached locally in cloudlog see the eqsl info on the wiki
